src/lekcja8_zadanie1_firma_szkoleniowa/Student.java

```diff
@@ -0,0 +1,24 @@
+package lekcja8_zadanie1_firma_szkoleniowa;
+
+public class Student extends Person {
```

Comment:
Brakuje tutaj metody ktora by dodawała ocene `addGrade`. Pokazalem jak to zrobic na ostatnich zajeciach

Reply:
Poprawione